### PR TITLE
MC-10542: update api doc for v2 delete environment

### PR DESCRIPTION
**Pull request by JamieWulala**

### CODE REVIEW

File: source/includes/administration/_environments.md [hunk 1]
```diff
@@ -244,10 +243,12 @@ You will need the `Environments update` permission to execute this operation.
 
 `DELETE /environments/:id`
 
+Purge an environment and its underlying resources asynchronously. If such purge operation fail for the first time, the service account of this environment would be updated to `ERROR_PURGING`. It would force purge an environment an `ERROR_PURGING` sercice account.
```
Comment (TheGreatHegazi) [inline source/includes/administration/_environments.md]: "it would force purge an environment and `error_purging` service account"
service account typo ! 
This seems a bit unclear to me. I think it should probably say something like: 
if called on an environment in error purging state then it would force purge the service account.

File: source/includes/administration/_environments.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ Attributes | &nbsp;
 ```shell
 # Retrieve visible environment
 
-curl "https://cloudmc_endpoint/v1/environment/487a2745-bb8a-44bc-adb1-e3b048f6def2" \
+curl "https://cloudmc_endpoint/v2/environments/487a2745-bb8a-44bc-adb1-e3b048f6def2" \
```
Comment (TheGreatHegazi) [inline source/includes/administration/_environments.md:83]: this change seems odd are you sure its environments not environment?

**Review by TheGreatHegazi — COMMENTED** [review]
looks good to me other than the comments

File: source/includes/administration/_environments.md (hunk 2 above)
Comment (JamieWulala) [reply source/includes/administration/_environments.md:83]: yes, I've tested it

File: source/includes/administration/_environments.md (hunk 1 above)
Comment (JamieWulala) [reply source/includes/administration/_environments.md]: updated

File: source/includes/administration/_environments.md [hunk 3]
```diff
@@ -243,7 +243,7 @@ You will need the `Environments update` permission to execute this operation.
 
 `DELETE /environments/:id`
 
-Purge an environment and its underlying resources asynchronously. If such purge operation fail for the first time, the service account of this environment would be updated to `ERROR_PURGING`. It would force purge an environment an `ERROR_PURGING` sercice account.
+Purge an environment and its underlying resources asynchronously. If such purge operation fail for the first time, the service account of this environment would be updated to `ERROR_PURGING`. If the purging an environment with `ERROR_PURGING` service account, it would force purge the environment.
```
Comment (TheGreatHegazi) [inline source/includes/administration/_environments.md]: if "the" purging an environment  
remove the "the" here please

**Review by TheGreatHegazi — APPROVED** [review]
Looks Good to me !

File: source/includes/administration/_environments.md [hunk 4]
```diff
@@ -244,10 +243,12 @@ You will need the `Environments update` permission to execute this operation.
 
 `DELETE /environments/:id`
 
+Purge an environment and its underlying resources asynchronously. If such purge operation fail for the first time, the service account of this environment would be updated to `ERROR_PURGING`. If the purging an environment with `ERROR_PURGING` service account, it would force purge the environment.
```
Comment (jasondias9) [inline source/includes/administration/_environments.md]: We shouldn't be referring to service accounts in the API doc. 
Lets rephrase it together shortly

**Review by jasondias9 — APPROVED** [review]
